### PR TITLE
feat: tauri version

### PR DESCRIPTION
**Pull request by sekwah41**
This is to implement #282 

Though needs a lot of clean up atm. Please check out the builds at https://github.com/sekwah41/pomatez/releases/tag/v1.5.0

though I will need to update this all to the right versions and links.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "image": "ubuntu:22.04" }
```
Comment (sekwah41) [inline .devcontainer/devcontainer.json:1]: This is just to test building on linux similar to the pipeline to see what missing deps there are

**Review by roldanjr — APPROVED** [review]
This looks good to me, merge it anytime you want @sekwah41. Thank you! 🙇🏼

**Comment by sekwah41** [comment]
I was going to add a progress bar for downloading though for windows and mac it would have been far too fast to download so I decided against it and added a notification just saying it'll take a moment then install.

**Comment by sekwah41** [comment]
Looking at the builds, it may be good to rename these to clearly state what platform they are for so that less experienced users do not need to look for the extensions though I will do that in a future update.

Ive got an old MacBook I can test on but it's an old 2012 one. I should have a new MacBook arriving in the next week or two to properly test on so will look to fix most of the issues then.

**Pull request merged**